java/ql/src/semmle/code/java/security/ExternalAPIs.qll

```diff
@@ -0,0 +1,146 @@
+/**
+ * Definitions for reasoning about untrusted data used in APIs defined outside the
+ * database.
+ */
+
+import java
+import semmle.code.java.dataflow.FlowSources
+import semmle.code.java.dataflow.TaintTracking
+
+module ExternalAPIs {
```

Comment:
```suggestion
```
Perhaps just drop this additional module layer. If it stays it requires qldoc, and it's not clear what it adds beyond the implicit file-level module.